css/index.css

```diff
@@ -909,21 +1056,50 @@ label {
     margin: auto;
   }
 
-  .article1div p,
-  .article2div p,
-  .article3div p,
-  .article4div p {
+  .popup_articlediv p {
+    color: #344563;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+  }
+
+  .popup_section .popup_article {
+    display: flex;
+    justify-content: center;
+    padding: 5px 0;
+    gap: 35px;
+    width: 85%;
+    margin: auto;
+    border-radius: 20px;
+    margin-top: 20px;
+  }
+
+  .popup_articlediv {
+    width: 90%;
+    margin: auto;
+    padding-top: 20px;
+  }
+
+  .footer_btn img {
+    width: 14px;
+    height: 17px;
+  }
+
+  .popup_article .figure2 img {
+    height: 582px;
+    object-fit: contain;
+    object-position: center;
+  }
```

Comment:
- [ ] Having made a great effort working on your pop-up modal, I must commend you that you have done a wonderful job 👏 👏 👏 , currently, your pop-up works really fine in the `mobile version`, but I notice in the `desktop version` the images are taking most part of the modal page as we have it in the Figma design, I would kindly suggest we stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible.


- [ ] Also I would suggest you increase the `text` as we have it `desktop` version as shown in the Figma design below

Your Design 👇 

![emma-pop](https://user-images.githubusercontent.com/78634841/209080269-8354f7a4-e204-4e17-b1ad-ac6bc19f5622.png)

Figma Design 👇 

<img src="https://user-images.githubusercontent.com/17243325/208959934-1e193ee9-09ff-4f6f-86d5-940a7ef55fd7.png">